feature_engine/creation/math_features.py

```diff
@@ -139,8 +193,19 @@ def __init__(
         new_variables_names: Optional[List[str]] = None,
         missing_values: str = "raise",
         drop_original: bool = False,
+        ddof: Union[int, float] = 1,
     ) -> None:
 
+        # casting input parameter func to a list
```

Comment:
for compatibility with sklearn, we can't modify the parameters that the user enters at init. They need to remain the same. Anything that needs changing, needs to happen in the fit and be added as a new attribute followed by _ if necessary at all.